drafts/developer-guide/pages/data-privacy/data-privacy-console.adoc

```diff
@@ -0,0 +1,58 @@
+include::ROOT:partial$variables.adoc[]
+
+= Data Privacy Console
```

Comment:
Should we provide a clearer separation of the two perspectives on the application -- one is for the users to exercise their privacy-related requests and the other one is for the DPO to review user-provided requests?

For me, currently, the document blurs the difference between them. Perhaps consider splitting them into separate documents? The end-user-facing interface will most likely end up in the end-user guide instead of the integrator guide.

Reply:
Should we provide more instructions on how to perform a specific privacy-related action, such as requesting data deletion?